exercises/retree/canonical-data.json

```
@@ -0,0 +1,57 @@
+{
+    "exercise": "retree",
+    "version": "1.0.0",
+    "cases": [
+      {
+        "description": "Empty tree",
+        "property": "treeFromTraversals",
+        "input": {
+          "preorder": [],
+          "inorder": []
+        },
+        "expected": {}
+      },
+      {
+        "description": "Tree with one item",
+        "property": "treeFromTraversals",
+        "input": {
+            "preorder": ["a"],
+            "inorder": ["a"]
+        },
+        "expected": {"v":"a", "l": {}, "r": {}}
+      },
+      {
+        "description": "Core functionality with realistic size traversal",
```

Maybe simplify this to:

```suggestion
        "description": "Tree with many items",
```

or 

```suggestion
        "description": "Large tree",
```